share/zm_rpcapi

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# PROVIDE: zm_rpcapi
+# REQUIRE: NETWORKING mysql postgresql
```

Comment:
What happens if not both of the database engines are installed?

Reply:
Regarding the meaning of REQUIRE I quote https://www.freebsd.org/cgi/man.cgi?rcorder(8)#end:

> The "REQUIRE" keyword is misleading: It doesn't describe which daemons have to be running before a script will be started. It describes which scripts must be placed before it in the dependency ordering. For example, if your script has a "REQUIRE" on "named", it means the script must be placed after the "named" script in the dependency ordering, not necessarily that it requires named(8) to be started or enabled.

My idea for dealing with unavailable database engines is described in #369. I believe, thought I haven't thoroughly tested it, that the service script part of #369 is implemented automagically in these scripts by virtue of the rc.d framework.